nft/service_test.go

```diff
@@ -280,7 +281,7 @@ func Test_getBundledHash(t *testing.T) {
 		byteSliceToByteArray32(common.FromHex("0xed58f4a0d0c76770c81d2b1cc035413edebb567f5c006160596dc73b9297a9cc")),
 	}
 
-	bh := common.FromHex("0xee49e1ca6aa1204cfb571094ce14ab254e5185005cbee3f26af9afd3140ac12d")
+	bh := common.FromHex("0xbbf0c526070f3b823eb995aa31f2ac9635532a0b010ff27183be7bf25b3f002a")
```

Comment:
Since we changed from using keccak to using blake2b the hash changed. In any case, this code most likely will not be used once we move all pools to the chain.